.travis.yml

```diff
@@ -33,6 +33,8 @@ script:
   - npm run lint-ci
   - npm run build-dll
   - |
-    if [ "$TRAVIS_OS_NAME" == "osx" ]; then 
-      npm run package-all;
+    if [ $TRAVIS_OS_NAME == "osx" ]; then 
+      rm -f release/auth.md
+      rm -f release/publishFilesList.txt
```

Comment:
see previous comment

Reply:
This file does exist once package-all has completed its run.

Reply:
ok